README.md

```diff
@@ -10,7 +10,8 @@ Good luck and happy hunting :tada:!
 
 | Company Name | Position | Location | Visa Support | Notes |
 |--------------|----------|----------|--------------|-------|
-| [Apple](https://jobs.apple.com/en-us/details/200253195/software-engineering-internship?team=STDNT) | SWE | Multiple locations | | Not sure if this is for Winter. |
+| [Apple](https://jobs.apple.com/en-us/details/200253195/software-engineering-internship?team=STDNT) | SWE | Multiple locations | | Not sure if this is for Winter. Edit: It is for fall/winter. |
```

Comment:
Oh in this case, just remove the notes and write something like `For Fall 2021 and Winter 2022`